### ISSUE TITLE
SUGGESTION: add to Pkgfile: md5sums array

### ISSUE THREAD

**Issue by peter1000** [NuTyX/cards#62]
I'm planning to implement this with the **complete rewrite** of pgmk2.in.

Similar idea to Arch [PKGBUILD](https://www.archlinux.org/pacman/PKGBUILD.5.html)

```
 md5sums (array)

    This array contains an MD5 hash for every source file specified in the source array (in the same order). makepkg will use this to verify source file integrity during subsequent builds. If SKIP is put in the array in place of a normal hash, the integrity check for that source file will be skipped. 
```

Not sure if to support this anything else needs to be changed in cards.

Anyway what I'm doing is: if the md5sums array is missing or has not all entries I generate one internally with all set to SKIP so it will be backwards compatible.

**Comment by peter1000** [comment]
Already done in my rewrite of pkgmk2.in - not yet uploaded.

**Comment by tnut** [comment]
Will look at it, I'm doing a pause with cards, for me it should always
works, means I very slow on devel

2016-04-06 13:31 GMT+02:00 peter1000 <notifications@github.com>:

**Comment by peter1000** [comment]
Will let you know when I have finished and uploaded: I also wrote already about 200 tests for the pkgmk.in so that unexpected results should be not so common in the future.

**Comment by peter1000** [comment]
Just a question: I implemented a `md5checksum array` for the Pkgfile - similar to what arch linux uses.

QUESTION: do we still need to write a separate `apr.x86_64.md5sum` file for something else.

May plan is that `pkgmk -um,  --update-md5sum       update md5sum` directly updates the Pkgfile array (this is what Arch linux does).

Can I skip the additional file or does card need it for something else - actually it can always be extracted also from the Pkgfile: similar to the code I wrote for the makedepends array. https://github.com/NuTyX/cards/blob/master/src/cards_depends.cpp#L130

Any thoughts?
